integration/docker/main_test.go

```diff
@@ -62,6 +62,20 @@ func TestIntegration(t *testing.T) {
 		}
 	}
 
+	// we need to check that processes like hypervisor, shim and proxy are not running
+	hypervisorPath := KataConfig.Hypervisor[DefaultHypervisor].Path
+	proxyPath := KataConfig.Proxy[DefaultProxy].Path
+	shimPath := KataConfig.Shim[DefaultShim].Path
+	generalProcesses := []string{hypervisorPath, proxyPath, shimPath}
+
+	for _, j := range generalProcesses {
+		cmd := NewCommand("pgrep", "-f", j)
```

Comment:
Apparently, as go is sort of mostly OS neutral, and there is no standard way to do this, invoking the commands directly is probably fine :-) I know normally we'd recommend seeking out the appropriate golang library.